lib/pacer/pacers.go

```diff
@@ -219,11 +219,23 @@ type GoogleDrive struct {
 	limiter  *rate.Limiter // rate limiter for the minSleep
 }
 
+// Linkbox is a specialized pacer for Linkbox
+type Linkbox struct {
```

Comment:
Why did you make a special Linkbox pacer? I think the generic one covers it doesn't it?

Reply:
We need a new pacer for the sake of having our own calculation function. Without this linkbox backend behavior leads to very unstable work. We could receive an empty directory despite putting files in it if we try to list it just after putting these files in it. Sometimes we need to check that the "not found error" is really caused by the absence of the file. This is the reason for adding retry for some functions, not only API calls: the response could have no errors but an empty array. Without these tricks, it is a very rare that all tests pass.

Reply:
Your pacer seems to set a flat retry of 200ms which is not best practice when retrying errors.

Did you try the default pacer with a minSleep of 200ms? That will be just about equivalent and will have the advantage that it will do exponential backoff if you allow it.

Reply:
As I understand minSleep in standard pacer works only if retry is triggered. My approach is to make a delay on every API call.

Reply:
No, you get minSleep every call to the pacer.

Reply:
I see. I changed pacer to standard and deleted my own. Thanks.